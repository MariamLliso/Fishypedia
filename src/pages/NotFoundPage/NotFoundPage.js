import styled from "styled-components";
import theme from "../../theme/theme";

const NotFoundStyle = styled.div`
  margin: 120px 20px 20px 20px;
  color: ${theme.brownDark};
  padding: 10px;
  background-color: ${theme.white};
  border-radius: 10px;

  h1 {
    text-align: center;
  }
`;

const NotFoundPage = () => {
  return (
    <NotFoundStyle>
      <h1>404</h1>
      <h2>It seems that you have not found any fishes...</h2>
    </NotFoundStyle>
  );
};
export default NotFoundPage;
