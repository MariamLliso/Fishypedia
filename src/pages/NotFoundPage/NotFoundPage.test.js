import TestRenderer from "react-test-renderer";
import NotFoundPage from "./NotFoundPage";

describe("Given a NotFoundPage component", () => {
  describe("When its rendered", () => {
    test("Then it should always match this snapshot", () => {
      const renderedNotFoundPage = TestRenderer.create(<NotFoundPage />);

      expect(renderedNotFoundPage).toMatchSnapshot();
    });
  });
});
