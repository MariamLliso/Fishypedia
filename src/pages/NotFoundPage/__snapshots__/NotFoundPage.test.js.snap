// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a NotFoundPage component When its rendered Then it should always match this snapshot 1`] = `
<div
  className="sc-bczRLJ cdtiOd"
>
  <h1>
    404
  </h1>
  <h2>
    It seems that you have not found any fishes...
  </h2>
</div>
`;
